src/main/java/de/themoep/inventorygui/InventoryGui.java

```diff
@@ -144,7 +144,13 @@ public class InventoryGui implements Listener {
             } catch (IllegalArgumentException | IncompatibleClassChangeError ignored) {}
         }
         if (clickSound == null) {
-            for (Sound sound : Sound.values()) {
+            Sound[] sounds;
+            try {
+                sounds = (Sound[]) Sound.class.getDeclaredMethod("values").invoke(null);
+            } catch (Exception e) {
+                throw new RuntimeException(e);
```

Comment:
Thank you for your PR, please do not thrown an exception here though as the sound will be set to the latest sound key further down if it wasn't found yet anyways! No need to prevent the whole GUI from working due to that.